src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -86,14 +112,22 @@ public GetCredentialCommand() : base()
         /// </summary>
         protected override void BeginProcessing()
         {
-            if (!string.IsNullOrEmpty(this.Message))
+           string caption;
+            if(!string.IsNullOrEmpty(this.Title)) {
```

Comment:
Looks like this code based uses a space between the `if` keyword and the opening paren. To be consistent with the rest of the code, you should add the space.